coding-concepts.js

```diff
@@ -15,40 +15,40 @@ const colors = ["tangerine", "magenta", "lilac", "daffodil"]
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: The length of the string's value (10)
+// b) Verify and explain:10 The .length request is asking for the value of the characters in the string.
```

Comment:
.length is going to ask for the number of values in a string including the space(_), not the values themselves